pyrene/src/components/MultiSelect/types.ts

```diff
@@ -1,5 +1,8 @@
+import { IconProps } from '../Icon/Icon';
+
 export interface Option {
+  iconProps?: IconProps,
   label: string,
-  invalid: boolean,
-  value: string | number,
+  value: string,
```

Comment:
Is there a reason why `number` is not accepted as a possible type for value?

Reply:
Yes, this is due to the `react-select` typing (npm dependency).